Updating Dockerfiles to remove use of APK

seanmcilvenna opened the pull request:
APK fails to run in other environments because of proxy rules, and is mostly unnecessary for these builds anyways.
Converted PatientsToQuery to use Ubuntu with APT instead, which is whitelisted in other environments already.

seanmcilvenna committed:
no message

edward-miller-lcg reviewed (commented):
I'm worried about removing:

```
ENV \
    DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
    LC_ALL=en_US.UTF-8 \
    LANG=en_US.UTF-8

```
When this isn't there, it tends to break Entity Framework. Alpine removes this that breaks EF. @amphillipsLGC

seanmcilvenna commented:
I can re-add that ENV statement to them... That's not the one that would break anything. I just thought it was a copy-paste for the APK.

seanmcilvenna committed:
Re-adding ENV lines for Entity Framework

seanmcilvenna commented:
@edward-miller-lcg I've re-added those ENV lines

seanmcilvenna committed:
Updating pipelines to only trigger for PRs that modify the service's code
Removing the "web" pipelines that are duplicate to "demo"
Removing the "shared" pipelines because all the pipelines build "shared" as part of each service

seanmcilvenna committed:
Patient list pipelines no longer valid. Duplicates of Census

seanmcilvenna committed:
Updating pipelines to build Docker images from a context of the `Java` directory to reduce the amount of files needing to be copied into docker context for java projects
Updating measure eval pipelines to build Java

(cherry picked from commit 039c1f617f5a1c74f948e6b3c81e867d017ca149)

smailliwcs committed:
Fix `measureeval` Dockerfile

Expect a build context of Java rather than Java/measureeval.

seanmcilvenna committed:
Merge branch 'fix-measureeval-dockerfile' into docker-dependencies

seanmcilvenna committed:
Merge branch 'dev' into docker-dependencies

seanmcilvenna committed:
Adding .Domain into consideration for the CI and Dockerfile

seanmcilvenna committed:
Basing Normalization service on Ubuntu image to handle invariant culture issue
